python/ray/ml/tests/test_torch_predictor.py

```diff
@@ -45,6 +45,20 @@ def test_predict():
     assert predictions.to_numpy().flatten().tolist() == [4, 8, 12]
 
 
+def test_predict_array_output():
+    """Tests if predictor works if model outputs an array instead of single value."""
+
+    predictor = TorchPredictor(model=model, preprocessor=preprocessor)
```

Comment:
Nit: this is definitely out of scope for this PR, but it might be better to define `model` and `preprocessor` in the test because:
* Less confusion when reading the tests since all of the code is visible from within each test.
* Less chance of sharing state between tests, which creates unwanted dependencies between them.

[Related advice](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#why-5) from .NET's unit testing best practices.

Reply:
Updated!